### PR TITLE
Upgrade for zeitgeistpm/sdk-v0.6.5

### PR DESCRIPTION
**Pull request by saboonikhil**
- Add links to argument types
- Update code snippets
- Sync with latest sdk changes

### CODE REVIEW

File: docs/build/sdk/indexs.md [hunk 1]
```diff
@@ -26,41 +26,62 @@ const res = await sdk.models.getAllMarketIds();
...
+      { name: `phala` },
+      { name: `robonomics` },
+      { name: `kilt` },
+    ],
+    slug: `kusama-derby-example`,
+    description: `example description`,
+    question: `who will win?`,
+  },
+  mdm: { authorized: `dE3pPiRvdKqPD5bUDBu3Xpi83McE3Zf3UG8CbhWBQfvUywd7U` },
+  swapFee: `10`,
```
Comment (maltekliemann) [inline docs/build/sdk/indexs.md:52]: If you use `10`, then you've a swap fee of `1e-09` percent, which is probably not what you want.

```suggestion
  swapFee: `1_000_000_000`,
```

File: docs/build/sdk/market.md [hunk 2]
```diff
@@ -82,21 +82,74 @@ const res = market.getDisputes();
...
+Buy complete sets and deploy a pool with specified liquidity for a market.
+
+```typescript
+const market = await sdk.models.fetchMarketData(marketId);
+
+const signer = util.signerFromSeed(`//Alice`);
+
+const poolId = await market.deploySwapPoolAndAdditionalLiquidity(
+    signer,
+    `10`,
```
Comment (maltekliemann) [inline docs/build/sdk/market.md:129]: If you use `10`, then you've a swap fee of `1e-09` percent, which is probably not what you want.

```suggestion
    `1_000_000_000`,
```

**Review by maltekliemann — CHANGES_REQUESTED** [review]
Although it would technically work to use `10` as swap fee, this doesn't seem like a realistic value. It should probably be `1_000_000_000` or `100_000_000`.

Other than that, two general comments:

- The indentation is inconsisent throughout the SDK documentation. Sometimes it's two spaces per indent, sometimes four.
- I think these examples would be easier to read if all numbers used a thousands separator, i.e. `10_000_000_000` instead of `10000000000`.

Also, would it be possible to define a `BASE` object in the SDK that people can use? It seems very timesome and frankly very error-prone to keep using explicit numbers.

**Comment by saboonikhil** [comment]
> Although it would technically work to use `10` as swap fee, this doesn't seem like a realistic value. It should probably be `1_000_000_000` or `100_000_000`.
> 
> Other than that, two general comments:
> 
> * The indentation is inconsisent throughout the SDK documentation. Sometimes it's two spaces per indent, sometimes four.
> * I think these examples would be easier to read if all numbers used a thousands separator, i.e. `10_000_000_000` instead of `10000000000`.
> 
> Also, would it be possible to define a `BASE` object in the SDK that people can use? It seems very timesome and frankly very error-prone to keep using explicit numbers.

I don't think the typescript data type `string` allows much.

We can work on the `BASE` object post discussion with the frontend team.

- If thousands separator is used, the error is thrown : `Struct: failed on swap_fee: Compact<u128>:: Invalid character`
- If `10,000,000,000` is used, the error looks like : `Struct: failed on swap_fee: Compact<u128>:: String should not contain decimal points or scientific notation`
- If `10` is used as `swapFee`, the pool is created with following properties:
Screenshot 2022-08-04 at 7 30 13 PM

**Comment by maltekliemann** [comment]
Nevertheless, I would not use `10` as swap fee (for instructive purposes).

Regarding the thousands separators, why are you guys using strings to communicate with polkadot-js?

**Comment by saboonikhil** [comment]
> 

What value would you recommend as an example for `swapFee`?

String? Just to keep it consistent with the [polkadot-js apps](https://polkadot.js.org/apps). They don't support thousands separators. Probably, another reason why it could potentially add to the confusion when combining using the SDK with just the usual polkadot-js api, as you'll end up with 2 different units there.

**Comment by maltekliemann** [comment]
I would expect numbers to be numbers and not strings, and so on. I would find it quite bothersome if I had to convert numbers to strings everytime I do an SDK call. Did I get this right, this is a restriction imposed by polkadot-js?

**Review by maltekliemann — APPROVED** [review]
:thumbsup:

**Comment by saboonikhil** [comment]
Thank you very much!

**Pull request merged**